packages/react-router-dom/server.tsx

```diff
@@ -224,7 +225,10 @@ export function createStaticHandler(
 
 export function createStaticRouter(
   routes: RouteObject[],
-  context: StaticHandlerContext
+  context: StaticHandlerContext,
+  opts?: {
```

Comment:
This doesn't seem to be used?

Reply:
ahh good catch, leftover from a prior API change